stacker/config/__init__.py

```diff
@@ -221,9 +224,12 @@ def validate_parameters(self, data, value):
         if value:
             stack_name = data['name']
             raise ValidationError(
-                    "Stack definition %s contains deprecated "
-                    "'parameters', rather than 'variables'. Please "
-                    "update your config." % stack_name)
+                    "DEPRECATION: Stack definition %s contains "
```

Comment:
Should this be `DEPRECATION WARNING` to be consistent with the others, or did you mean for this to be different on purpose?